### PR TITLE
Doc update for OpenShift installation settings

### PR DESCRIPTION
**Pull request by artberger**
# Description

Fixes OpenShift installation instructions.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [N/A] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [N/A] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [N/A] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [N/A] I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/5619

### CODE REVIEW

File: docs/content/installation/gateway/kubernetes/_index.md [hunk 1]
```diff
@@ -13,16 +13,23 @@ Minimum required Kubernetes is 1.11.x. For older versions see our [release suppo
 
 ---
 
-## Installing the Gloo Edge on Kubernetes
+## Before you begin
+
+1. Make sure that you prepared your Kubernetes cluster according to the [platform configuration for Kubernetes instructions]({{% versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/" %}}).
```
Comment (Rachael-Graham) [inline docs/content/installation/gateway/kubernetes/_index.md:18]: same comments in this file as above

File: docs/content/installation/gateway/kubernetes/_index.md [hunk 2]
```diff
@@ -186,31 +193,28 @@ NOT opening the listener ports when there are no listeners (routes) is by design
 
 ## Uninstall {#uninstall}
```
Comment (Rachael-Graham) [inline docs/content/installation/gateway/kubernetes/_index.md:194]: what is this, i wonder

File: docs/content/installation/gateway/kubernetes/_index.md (hunk 1 above)
Comment (artberger) [reply docs/content/installation/gateway/kubernetes/_index.md:18]:
```suggestion
1. Make sure that you prepared your Kubernetes cluster according to the [instructions for platform configuration]({{% versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/" %}}).
```

File: docs/content/installation/gateway/kubernetes/_index.md [hunk 3]
```diff
@@ -13,16 +13,23 @@ Minimum required Kubernetes is 1.11.x. For older versions see our [release suppo
...
+   {{% notice note %}}
+   Pay attention to provider-specific information in the setup guide. For example, [OpenShift]({{< versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/#openshift" >}}) requires stricter multi-tenant support, so the setup guide includes an example Helm chart `values.yaml` file that you must supply while installing Gloo Edge Enterprise.
+   {{% /notice %}}
+2. Get your Gloo Edge Enterprise license key. If you don't have one already, you may request a trial license key [here](https://www.solo.io/products/gloo/#enterprise-trial).
+   {{% notice info %}}
+   Each trial license key is typically valid for **30 days**. You can request a new key if your current key has expired. When the license key expires, you can request a new license key by contacting your Account Representative or filling out [this form](https://lp.solo.io/request-trial). You must provide the license key during the installation process. When you install Gloo Edge, a Kubernetes secret is created to store the license key. When the key is about to expire, see [Updating Enterprise Licenses]({{< versioned_link_path fromRoot="/operations/updating_license/" >}}).
```
Comment (artberger) [inline docs/content/installation/gateway/kubernetes/_index.md:24]:
```suggestion
   You must provide the license key during the installation process. When you install Gloo Edge, a Kubernetes secret is created to store the license key. Note that each trial license key is typically valid for **30 days**. When the license key expires, you can request a new license key by contacting your Account Representative or filling out [this form](https://lp.solo.io/request-trial). For more information, see [Updating Enterprise Licenses]({{< versioned_link_path fromRoot="/operations/updating_license/" >}}).
```

**Review by Rachael-Graham — APPROVED** [review]

**Pull request merged**